Makefile

```diff
@@ -0,0 +1,36 @@
+SHELL=/bin/bash
+
+dep: godep reactdep
+
+godep:
+	glide up
+
+reactdep:
+	cd web/react && yarn
+
+build: gobuild reactbuild
+
+gobuild:
+	go build
+
+reactbuild:
+	cd web/react && yarn build
+
+test: gotest reacttest
+
+gotest:
+	go test ./...
+
+reacttest:
+	cd web/react && yarn test
+
+lint: golint
+
+golint:
```

Comment:
gometalinter is deprecated.
Perhaps in a later PR it is worth to try https://github.com/golangci/golangci-lint
After trying here, we could replace gometalinter in the back-end repo to not fall behind.

Reply:
good point, done